### PR TITLE
NOISSUE - Fix Bugs On Syncing With Docs

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
Fix bugs present when updating docs 

### Which issue(s) does this PR fix/relate to?
NOISSUE

### List any changes that modify/break current functionality
No major change

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### Notes
Linked to https://github.com/mainflux/docs/pull/138
https://github.com/mainflux/mainflux/pull/1768 and https://github.com/mainflux/mainflux/pull/1767 to be merged first and rebased with https://github.com/mainflux/mainflux/tree/clients

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,9 @@
 # https://digitalfortress.tech/tricks/creating-a-global-gitignore/
 
 build
+
+# tools
```
Comment (drasko) [inline .gitignore]: why are we ignoring all the tools? We would like maybe to develop those.

**Review by drasko — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (rodneyosodo) [reply .gitignore]: https://github.com/mainflux/mainflux/pull/1741#discussion_r1154588982

File: cmd/bootstrap/main.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/go-redis/redis/v8"
 	"github.com/jmoiron/sqlx"
 	"github.com/mainflux/mainflux/bootstrap"
-	"github.com/mainflux/mainflux/bootstrap/api"
+	api "github.com/mainflux/mainflux/bootstrap/api"
```
Comment (SammyOina) [inline cmd/bootstrap/main.go]:
```suggestion
	"github.com/mainflux/mainflux/bootstrap/api"
```

File: things/policies/api/http/responses.go [hunk 3]
```diff
@@ -16,8 +16,8 @@ var (
 )
 
 type policyRes struct {
-	policies.Policy
-	created bool
+	Policies []policies.Policy `json:"policies"`
+	created  bool
```
Comment (SammyOina) [inline things/policies/api/http/responses.go]: include a struct tag json

File: things/policies/page.go [hunk 4]
```diff
@@ -10,11 +10,11 @@ type Page struct {
 	Total    uint64
```
Comment (SammyOina) [inline things/policies/page.go]: include struct tags for other fields

File: users/clients/api/requests.go [hunk 5]
```diff
@@ -134,15 +145,13 @@ func (req updateClientOwnerReq) validate() error {
...
 }
 
-type updateClientCredentialsReq struct {
-	token     string
-	id        string
-	Identity  string `json:"identity,omitempty"`
-	OldSecret string `json:"old_secret,omitempty"`
-	NewSecret string `json:"new_secret,omitempty"`
+type updateClientIdentityReq struct {
+	token    string
```
Comment (SammyOina) [inline users/clients/api/requests.go]: add struct tags

File: users/groups/status.go [hunk 6]
```diff
@@ -7,8 +7,8 @@ type Status uint8
 
 // Possible Group status values
 const (
-	DisabledStatus Status = iota
-	EnabledStatus
+	EnabledStatus Status = iota
```
Comment (SammyOina) [inline users/groups/status.go:11]: add comments for exported const

File: api/openapi/users.yml [hunk 7]
```diff
@@ -21,33 +21,33 @@ tags:
   - name: Users
     description: Everything about your Users
     externalDocs:
-      description: Find out more
-      url: http://swagger.io
+      description: Find out more abour users
```
Comment (SammyOina) [inline api/openapi/users.yml]:
```suggestion
      description: Find out more about users
```

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: users/clients/api/requests.go (hunk 5 above)
Comment (rodneyosodo) [reply users/clients/api/requests.go]: `token` is not exported and it is decode from the header not request body

File: things/policies/api/http/responses.go (hunk 3 above)
Comment (rodneyosodo) [reply things/policies/api/http/responses.go]: we are not serializing `created`

File: docker/templates/users.tmpl [hunk 8]
```diff
@@ -1,9 +1,13 @@
-To: {{range $index, $v := .To}}{{if $index}},{{end}}{{$v}}{{end}}
-From: {{.From}}
-Subject: {{.Subject}}
-{{.Header}}
-You have initiated password reset.
-Follow the link below to reset password.
+Dear {{.User}},
```
Comment (dborovcanin) [inline docker/templates/users.tmpl:1]: Did you test the password reset email?

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: docker/templates/users.tmpl (hunk 8 above)
Comment (rodneyosodo) [reply docker/templates/users.tmpl:1]: Yes. It works but it works funny

**Comment by rodneyosodo** [comment]
CI is fixed by [`6e919ba` (#1781)](https://github.com/mainflux/mainflux/pull/1781/commits/6e919ba872434c328af544e22410687ea0a11490)

**Pull request merged**